src/constraint/spec.ts

```diff
@@ -264,6 +265,24 @@ export const SPEC_CONSTRAINTS: SpecConstraintModel[] = [
     }
   },
   {
+    name: 'omitAggregatePlotWithoutDimension',
+    description: 'Aggregate plots without dimension should be omitted',
+    properties: [Property.AGGREGATE, Property.AUTOCOUNT, Property.TYPE],
```

Comment:
Are these sufficient for checking `isDimension`?   I bet you haven't checked that.